backend/app/services/channels/dingtalk/handler.py

```
@@ -436,6 +457,60 @@ def _build_legacy_message_context(
+                    mime_type = "image/png"
+
+                base64_data = base64.b64encode(response.content).decode("utf-8")
+                images.append(
+                    {"mime_type": mime_type, "base64_data": base64_data}
+                )
+                self.logger.info(
+                    "[DingTalkHandler] Downloaded image: mime=%s, size=%d bytes",
+                    mime_type,
+                    len(response.content),
+                )
+            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)